### PR TITLE
Resolves #10

### LINKED ISSUE #10: Add README
We need to add a README, which would include:

- The rationale behind the Widget
- Some examples of how it should look
- Usage instructions
   - Basic usage
   - Customization (UI layout, theming)
   - Localization (depends on #7)

### PR DESCRIPTION
**Pull request by solid-danylosafonov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,63 @@
...
-from the package authors, and more.
+### Customization
+You can add widget floating above your app like that:
+```
+OverlayWidget(
+    alignment: alignment,
+    overlayWidget: HorizontalHelpWidget(),
+    child: MyApp(),
+)
+```
```
Comment (solid-yuriiprykhodko) [inline README.md]: Let's be a bit more verbose here.

Basic usage involves baking it into user's layout -- let's make it explicit.

I'd also say thet using the OverlayWidget is a case of basic usage -- make sure to describe how to position the widget using it.

As for customization -- we should be explaining how the users could make their own layout, reusing widgets/controllers from our code.

File: README.md [hunk 2]
```diff
@@ -1,39 +1,63 @@
...
 ## Getting started
+<h5 a><strong><code>pubspec.yaml</code></strong></h5>
 
-TODO: List prerequisites and provide or point to information on how to
-start using the package.
-
-## Usage
+``` yaml
+dependencies:
+    help_ukraine_widget: ^1.0.0
```
Comment (solid-yuriiprykhodko) [inline README.md]: We'll probably start with an unstable version. Besides -- we don't want to keep updating the version in the README each time we do a release.
```suggestion
    help_ukraine_widget: <latest version>
```

File: README.md [hunk 3]
```diff
@@ -1,39 +1,63 @@
...
+By installing this widget in your app, you will help to end the war in Ukraine.
+It will provide links for:
 
-TODO: List what your package can do. Maybe include images, gifs, or videos.
+* [donations](https://uahelp.monobank.ua/),
+* sharing this widget
+* and [other ways to support Ukraine](https://war.ukraine.ua/).
 
 ## Getting started
+<h5 a><strong><code>pubspec.yaml</code></strong></h5>
```
Comment (solid-yuriiprykhodko) [inline README.md]: HTML here is uncalled for, let's stick to plain Markdown wherever possible.
Besides, we could be a bit more verbose:
```suggestion
Add the package as a dependency to your __pubspec.yaml__:
```

**Review by solid-yuriiprykhodko — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1,39 +1,135 @@
...
+import 'package:help_ukraine_widget/help_ukraine_widget.dart';
+```
 
 ## Usage
 
-TODO: Include short and useful examples for package users. Add longer examples
-to `/example` folder. 
+### Basic usage
+Use `OverlayWidget` to float and align the widget above your app:
+```
```
Comment (solid-yuriiprykhodko) [inline README.md]:
```suggestion
``` dart
```

Comment (solid-danylosafonov) [reply README.md]: done

File: README.md [hunk 5]
```diff
@@ -1,39 +1,135 @@
...
-This README describes the package. If you publish this package to pub.dev,
-this README's contents appear on the landing page for your package.
+# Help Ukraine Widget for Flutter
+This is a port of [Help Ukraine Widget](https://helpukrainewinwidget.org/) -
+this widget helps to let users of your app know how they can help Ukraine
+from anywhere in the world.
 
-For information about how to write a good package README, see the guide for
-[writing package pages](https://dart.dev/guides/libraries/writing-package-pages). 
+### Some examples
```
Comment (solid-yuriiprykhodko) [inline README.md]:
```suggestion
### Showcase
```

File: README.md [hunk 6]
```diff
@@ -1,39 +1,135 @@
...
+final _controller = TraverseController(order);
 
-```dart
-const like = 'sample';
+HelpWidget(
+    controller: _controller;
+    // view with links to resources that help Ukraine
+    optionsView: ...
+    // transitional view between options and collapsed
+    mainView: ...
```
Comment (solid-yuriiprykhodko) [inline README.md]: Not really logical to describe it as "transitional" -- this is the view that the users see first as they open the page.

```suggestion
    // The default first view that the users see.
    mainView: ...
```

File: README.md [hunk 7]
```diff
@@ -1,39 +1,135 @@
...
 ## Usage
 
-TODO: Include short and useful examples for package users. Add longer examples
-to `/example` folder. 
+### Basic usage
+Use `OverlayWidget` to float and align the widget above your app:
+``` dart
+OverlayWidget(
+    // e.g. Alignment.bottomRight
+    alignment: alignment,
```
Comment (solid-yuriiprykhodko) [inline README.md]: Simpler: 

```suggestion
    alignment: Alignment.bottomRight,
```

File: README.md [hunk 8]
```diff
@@ -1,39 +1,134 @@
-<!-- 
-This README describes the package. If you publish this package to pub.dev,
-this README's contents appear on the landing page for your package.
+# Help Ukraine Widget for Flutter
+This is a port of [Help Ukraine Widget](https://helpukrainewinwidget.org/) -
```
Comment (solid-yuriiprykhodko) [inline README.md:5]: Style solid badge:

```suggestion
[![style: solid](https://img.shields.io/badge/style-solid-orange)](https://pub.dev/packages/solid_lints)

This is a port of [Help Ukraine Widget](https://helpukrainewinwidget.org/) -
```

File: README.md [hunk 9]
```diff
@@ -1,39 +1,136 @@
...
-TODO: Put a short description of the package here that helps potential users
-know whether this package might be useful for them.
+<details open><summary>First widget variation</summary>
+
+![](gifs/widget1.gif)
+</details>
+
+<details><summary>Second widget variation</summary>
+    
+![](gifs/widget2.gif)
```
Comment (solid-yuriiprykhodko) [inline README.md]: image

Comment (solid-yuriiprykhodko) [reply README.md]: Highly suggest using markdownlint

Comment (solid-danylosafonov) [reply README.md]: added

File: README.md [hunk 10]
```diff
@@ -1,39 +1,149 @@
...
+
+`XMark` - flexible cross icon.
+
+### Controllers
+
+`TraverseController` is used to track current view and direction of last transition.
+
+### Helpers
+
+`defaultOptionsList` - list of default link buttons for external resources.
```
Comment (solid-yuriiprykhodko) [inline README.md]: Let's probably scrap this.

High chance to get this out of sync with real implementations. Much more reliable to redirect to autogenerated API docs as soon as we're out on pub.dev.

**Comment by solid-danylosafonov** [comment]
empty link is temporary

File: README.md [hunk 11]
```diff
@@ -1,39 +1,97 @@
...
+    // using the app, but still exists on the screen
+    collapsedView: ...
+    // axis of transition animation between views
+    axis: ...
+)
 ```
 
-## Additional information
+To make it easier we provide some [classes](#).
+
```
Comment (solid-yuriiprykhodko) [inline README.md]: Let's not leave placeholder text in -- we'll add it when we get to it.
```suggestion
```

**Review by solid-yuriiprykhodko — APPROVED** [review]
Alright, looking sleek for the first release!

**Pull request merged**